src/python/turicreate/toolkits/object_detector/object_detector.py

```diff
@@ -345,7 +352,27 @@ def create(dataset, annotations=None, feature=None, model='darknet-yolo',
     else:
         num_iterations = max_iterations
 
+
+    #create tensorflow model here
+    if USE_CPP:
+        import turicreate.toolkits.libtctensorflow
+
+        tf_config = {
+            'batch_size' : batch_size,
+            'grid_height': params['grid_shape'][0],
+            'grid_width': params['grid_shape'][1],
+            'max_iterations': num_iterations,
+            'mlmodel_path' : params['mlmodel_path'],
+            'classes' : classes
+
+        }
+        model = _tc.extensions.object_detector()
+        model.train(data=dataset, annotations_column_name=annotations, image_column_name=feature, options=tf_config)
+        return ObjectDetector_beta(model_proxy=model, name="object_detector")
```

Comment:
Why can't we move this earlier and avoid more of the `if not USE_CPP` cases above?